packages/shared/lib/services/file/local.service.ts

```diff
@@ -184,8 +184,10 @@ class LocalFileService {
             return;
         });
 
+        const timestamp = Math.floor(new Date().getTime() / 1000).toString();
+        const filename = `${timestamp}-nango-integrations.zip`;
         res.setHeader('Content-Type', 'application/zip');
-        res.setHeader('Content-Disposition', `attachment; filename=nango-integrations.zip`);
+        res.setHeader('Content-Disposition', `attachment; filename=${filename}`);
```

Comment:
Did this work for you? I downloaded a file and the name was still `nango-integrations.zip`. Editing the name in the [FlowPage](https://github.com/NangoHQ/nango/blob/master/packages/webapp/src/pages/Integration/FlowPage.tsx#L93) worked as expected h/e